pkg/dockerregistry/server/auth.go

```diff
@@ -53,7 +53,7 @@ const (
 // RegistryClient encapsulates getting access to the OpenShift API.
 type RegistryClient interface {
 	// Clients return the authenticated clients to use with the server.
-	Clients() (client.Interface, kclientset.Interface, error)
+	Clients() (client.Interface, kcoreclient.LimitRangesGetter, error)
```

Comment:
I made this change to remove `fake.NewSimpleClientset()` from tests. Otherwise there is no reason to use specific interface.